rust/sql/src/postgres/reader.rs

```diff
@@ -0,0 +1,938 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This is pretty cool (a partial implementation of hte postgres line protocol and conversions to/from arrow). I wonder if we could leverage something like https://github.com/sfackler/rust-postgres rather than taking on a new independent implementation

Reply:
The reader and writer are built on rust-postgres

Reply:
got it 👍